provider/machinepool/hcp/aws_node_pool.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 type AWSNodePool struct {
-	InstanceType    types.String `tfsdk:"instance_type"`
-	InstanceProfile types.String `tfsdk:"instance_profile"`
-	Tags            types.Map    `tfsdk:"tags"`
+	InstanceType               types.String `tfsdk:"instance_type"`
+	InstanceProfile            types.String `tfsdk:"instance_profile"`
+	Tags                       types.Map    `tfsdk:"tags"`
+	AdditionalSecurityGroupIds types.List   `tfsdk:"aws_additional_security_group_ids"`
```

Comment:
I don't think this requires the aws_ prefix as it's already within the aws node pool field. On classic we needed it because it was on the root structure so we needed it to show it is aws only

Reply:
Good point. Changed, thanks.